### DISCUSSION TITLE
drf-excel

### DISCUSSION THREAD

**Discussion by williln** [django-commons/membership#106] [Check-ins]
Hi @django-commons/drf-excel! I wanted to take a moment and check in with you about how the transition to Django Commons has been. 

- Have you run into any friction having your project in Django Commons, if so, can you describe them?
- Is there anything we can do to help your team or repo out?
- Do you have any questions or concerns for us?

If you'd like to chat with us privately, our email is [django-commons-admin@googlegroups.com](mailto:django-commons-admin@googlegroups.com). Thanks!

**Comment by browniebroke** [comment]
> - Have you run into any friction having your project in Django Commons, if so, can you describe them?

Not really, if anything some frictions was removed: code coverage report was fixed and releases are much easier to make. There hasn't ben a lot of bugs/changes to the project.

> - Is there anything we can do to help your team or repo out?

I think we're good so far.

> - Do you have any questions or concerns for us?

Can't think of anythng at the moment. Thanks

**Comment by FlipperPA** [comment]
Moving to Django Commons has been a big help, and a relief. It is how I met @browniebroke, who's become a maintainer. And I'm relived knowing that this package, which gets a fair amount of use, has a team supporting it instead of just me. Thanks for checking in, @williln!

**Comment by williln** [comment]
@FlipperPA @browniebroke This is all great to hear! Thanks for checking in! We'll check back in in a few months, but in the meantime, you know where to find us if you need us! 

>  if anything some frictions was removed: code coverage report was fixed and releases are much easier to make 

❤️

**Comment by tim-schilling** [comment]
Hi @django-commons/drf-excel, we're circling back around :dizzy: We want to ask [Lacey's original questions](https://github.com/orgs/django-commons/discussions/106#discussion-7490224) to see if there's any update. We're also looking into how can we improve the new contributor experience with the overall goal of helping find regular contributor and more maintainers for projects.
- [Meet and greets](https://github.com/orgs/django-commons/discussions/136)
- [Project introduction videos](https://github.com/orgs/django-commons/discussions/150)

If any of that sounds interesting to you, feel free to engage on the discussions or here.

**Comment by FlipperPA** [comment]
I'm happy to host a meet and greet, but we're in a pretty good space as we're a fairly stable package that does one thing fairly well. @browniebroke has been amazing with taking the ball and running with it, I can't say enough good things. If he ever needs a reference, I'm happy to provide one!

There are some open issues tagged `help wanted` if any new contributors want to start looking. Others have discussions because they aren't fully fleshed out, or perhaps beyond the scope of the features we want to provide, such as https://github.com/django-commons/drf-excel/issues/17

**Reply by tim-schilling** [reply]
That's a great state to be in! Thank you for the information. It's helping us to piece together how we should report/represent the community.

**Comment by williln** [comment]
Hi @django-commons/drf-excel -- this is just our semi-annual checkin. Let us know if there is anything Django Commons can do to help you or your project, or if there is anything else on your mind! Otherwise, happy coding 🌻

**Comment by FlipperPA** [comment]
Thanks @williln, things seem to be going smoothly. The package is stable, and with DRF upstream not really changing anymore, it feels like we're in a good place. Missed you at DCUS! You got a shout-out in my talk. :smile:

**Comment by tim-schilling** [comment]
Hi @django-commons/drf-excel this is just our semi-annual checkin. Let us know if there is anything Django Commons can do to help, or if there is anything else on your mind! Otherwise, happy coding 🥳

As an additional reminder, though not at all a suggestion/requirement. It may be worth looking through who has regularly contributed to your project to see if you'd like to elevate them to be a committer or a maintainer. I mention it because it's easy to forget to review the repositories community and helping people grow. Similarly, if you or anyone else wants to step back, that's fine too! If we can help coordinate, please let us know.

**Reply by tim-schilling** [reply]
Whoops, forgot some things. There are two other things:

1. Can the @django-commons/drf-excel-admins please share your emails with us? You can email us at [django-commons-admin@googlegroups.com](mailto:django-commons-admin@googlegroups.com) to avoid publishing publicly here.
2. We are working on a new policy around project abandonment. As a community organization, we need to have clear guidelines on when the community should pass the project onto someone else or signaling that the project is in a period of low activity. We'll share this new policy with you when we have it ready for comment. This will include a timeline for adoption. If you have questions, please let us know!

**Comment by FlipperPA** [comment]
All seems well! DRF will be around for a while, but most people looking for "shiny and new" seem to be trying django-ninja. I'm not sure how many new features this package will see, but we'll continue to keep it current with downstream dependencies.
